portfolio/src/main/java/com/google/sps/servlets/ServerDatesServlet.java

```diff
@@ -0,0 +1,43 @@
+package com.google.sps.servlets;
+
+import java.util.ArrayList;
```

Comment:
I know it's tedious (and you might find some tooling for it) but

https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing